Library/Homebrew/requirements/php_requirement.rb

```diff
@@ -0,0 +1,11 @@
+class PhpRequirement < Requirement
+  fatal true
+  satisfy { which "php" }
+
+  env do
+    if which("php") != "/usr/bin/php"
```

Comment:
It is first in my `$PATH`, but homebrew doesn't pick that up (it sets up a `$PATH` of its own, hence this hack).